package.json

```diff
@@ -45,12 +45,7 @@
     "twing:copy": "ncp static build",
     "twing:clean": "rimraf build dist",
     "twing:dist": "npm-run-all twing:clean storybook-build-twing:* twing:copy",
-    "update-version": "lerna version --no-push --no-git-tag-version",
-    "clean": "yarn twing:clean",
-    "copy": "yarn twing:copy",
-    "dist": "yarn twing:dist",
```

Comment:
Also one central `dist` running both systems' dist will also be useful. Having `yarn clean && yarn dist` will prepare everything in one line clean line